tests/full-tutorial.config.json

```diff
@@ -80,7 +80,7 @@
         "reference_inputs/ensembl69_hg19_annotations.v3.json"
     ],
     "reference.dgv_annotation": [
-        "reference_inputs/dgv_hg19_variants.tab"
+        "tests/data/mock_dgv_annotation.txt"
```

Comment:
this is still a tab delimited file right? if so it should be `.tab` to match all the other output/inputs

Reply:
Addressed in [960bd67](https://github.com/bcgsc/mavis/pull/336/commits/960bd67fdca774210a6cd5aac4c8c14bbe2a2a5f)